Labelbox/labelbox-python
# Move parser to the root of labelbox

Original location of the parser was in labelbox.data..... and that caused side-effect when importing it. I.e. some packages were loaded that tried to import optional libraries like geojson causing a `ModuleNotFoundError` exception

By moving the parser into the root folder, we make sure the import command does not search any other modules causing this issue

To test:

- checkout this branch
- `pip uninstall geojson`
- `pip3 install -e .`
- run `python`
- inside the python interpreter run `import labelbox as lb` - should see no exception

(but when doing the same on develop WILL see the exception)
